compiler/natives/src/net/http/fetch.go

```diff
@@ -115,8 +116,15 @@ func (t *fetchTransport) RoundTrip(req *Request) (*Response, error) {
 			}
 		},
 		func(reason *js.Object) {
+			var jsErr error
+			if reason != nil {
+				jsErr = fmt.Errorf("net/http: fetch failed: %s", reason.String())
+			} else {
+				jsErr = errors.New("net/http: fetch() failed")
```

Comment:
Why is one message "net/http: fetch" but the other "net/http: fetch()"? It'd be better to make them consistent.